doc/user_guide/getting_started.rst

```diff
@@ -7,216 +7,54 @@ Getting started
-
-            The ``toolkit`` and ``pylab_inline`` optional arguments are no
-            longer supported. In order to configure the toolkit use the
-            :ref:`hyperspy magic <magic-label>` after starting an Jupyter
-            notebook.
-
-
-Starting HyperSpy from the context menu
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This option is only available for Windows and in Linux when using Gnome.
-
-Windows
-"""""""
-
-In Windows it is possible to start HyperSpy from :menuselection:`Start Menu -->
-Programs --> HyperSpy`.
-
-Alternatively, one can start HyperSpy in any folder by pressing the :kbd:`right
-mouse button` or on a yellow folder icon or (in some cases) on the empty area
-of a folder, and choosing :menuselection:`HyperSpy qtconsole here` or
-:menuselection:`HyperSpy notebook here` from the context menu.
-
-
-.. figure::  images/windows_hyperspy_here.png
-   :align:   center
-   :width:   500
-
-   Starting hyperspy using the Windows context menu.
-
-
-Linux
-"""""
-
-If you are using GNOME in Linux, you can open a terminal in a folder by
-choosing :menuselection:`open terminal` in the file menu if
-:program:`nautilus-open-terminal` is installed in your system.
-
-Alternatively (and more conveniently), if you are using GNOME place `this
-<https://github.com/downloads/hyperspy/hyperspy/HyperSpy%20QtConsole%20here.sh>`_
-and `this
-<https://github.com/downloads/hyperspy/hyperspy/HyperSpy%20Notebook%20here.sh>`_
-in the :file:`~/.local/share/nautilus/scripts/` folder in your home directory
-(create it if it does not exists) and make them executable to get the
-:menuselection:`Scripts --> HyperSpy QtConsole Here` and
-:menuselection:`Scripts --> HyperSpy Notebook Here` entries in the context
-menu.
-
-
-.. figure::  images/hyperspy_here_gnome.png
-   :align:   center
-   :width:   500
-
-   Starting hyperspy using the Gnome nautilus context menu.
-
-.. _magic-label:
-
-HyperSpy IPython magic
-^^^^^^^^^^^^^^^^^^^^^^
-
-.. versionchanged:: 0.8.3
-
-HyperSpy provides an IPython magic in order to ease the task of setting up the
-typical interactive data analysis environment. The magic imports the HyperSpy API,
-`numpy <http://www.numpy.org/>`_, `matplotlib <http://matplotlib.org/>`_ and
-sets up the right toolkit. Before HyperSpy 0.8.1, all this was done
-automatically.  However, starting with IPython version 3.0, auto-importing and
-auto-configuring is discouraged (for very good reasons). In order to comply
-with the new recommendations without loosing convenience we supply the
-``hyperspy`` IPython magic that performs the same operations in a more explicit
-and transparent way. As of IPython version 4.0 and HyperSpy 0.8.3, HyperSpy has
-to be loaded before the magic becomes available:
+Typically you will need to `set up IPython for interactive plotting with
+matplotlib
+<http://ipython.readthedocs.org/en/stable/interactive/plotting.html>`_ using the
+```%matplotlib`` magic. So, typically, after starting IPython, you can import
```

Comment:
too many ticks before `%matplotlib` (i.e. one is displayed as well, as in <code>`%matplotlib</code>)